extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php

```diff
@@ -10,12 +10,11 @@ class ARecoveryEngineHooks {
 	 * @return bool
 	 */
 	public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
+
 		if ( !ARecoveryModule::isEnabled() ) {
-			return true;
+			$scripts .= F::app()->sendRequest( 'ARecoveryEngineApiController', 'getBootstrap' );
```

Comment:
What was the reason to change it? We add the SP bootstrap now everytime when recovery is not enabled.